lib/fog/azurerm/models/storage/blob.rb

```diff
@@ -5,8 +5,124 @@ class AzureRM
       # delete/destroy for Blob.
       class Blob < Fog::Model
         identity  :name
+        attribute :accept_ranges
+        attribute :cache_control
+        attribute :committed_block_count
         attribute :container_name
+        attribute :content_length
+        attribute :content_type
+        attribute :content_md5
+        attribute :content_encoding
+        attribute :content_language
+        attribute :content_disposition
+        attribute :copy_completion_time
+        attribute :copy_status
+        attribute :copy_status_description
+        attribute :copy_id
+        attribute :copy_progress
+        attribute :copy_source
+        attribute :etag
+        attribute :lease_duration
+        attribute :lease_state
+        attribute :lease_status
+        attribute :last_modified
         attribute :metadata
+        attribute :sequence_number
+        attribute :blob_type
+
+        def save(options = {})
+          requires :name
+          requires :container_name
+          if options.key?(:file_path)
+            merge_attributes(Blob.parse(service.upload_block_blob_from_file(container_name, name, options[:file_path], options)))
+          else
+            merge_attributes(Blob.parse(service.upload_block_blob_from_file(container_name, name, nil, options)))
+          end
+        end
+
+        alias create save
+
+        def destroy(options = {})
+          requires :name
+          requires :container_name
+          service.delete_blob container_name, name, options
+        end
+
+        def get_to_file(file_path, options = {})
+          requires :name
+          requires :container_name
+          merge_attributes(Blob.parse(service.download_blob_to_file(container_name, name, file_path, options)))
+        end
+
+        def get_properties(options = {})
+          requires :name
+          requires :container_name
+          merge_attributes(Blob.parse(service.get_blob_properties(container_name, name, options)))
+        end
+
+        def set_properties(options = {})
```

Comment:
We suggest to rename 'options' argument to 'properties', which is more intuitive here.